xbmc/settings/lib/SettingsManager.h

```diff
@@ -58,7 +58,7 @@ class CSettingsManager : public ISettingCreator, public ISettingControlCreator,
   CSettingsManager() = default;
   virtual ~CSettingsManager();
 
-  static const uint32_t Version = 1;
+  static const uint32_t Version = 2;
```

Comment:
This is just the current version (which will be written to `guisettings.xml`). One line further down is the minimum supported version which is still at `0` which is identical to not defining a version at all (which we didn't until a few weeks ago).